src/modules/privacy-idea-administration/privacy-idea-administration.controller.ts

```diff
@@ -149,4 +152,32 @@ export class PrivacyIdeaAdministrationController {
             );
         }
     }
+
+    @Post('verify')
+    @HttpCode(HttpStatus.CREATED)
+    @ApiCreatedResponse({ description: 'The token was successfully verified.' })
+    @ApiBadRequestResponse({ description: 'A username was not given or not found.' })
+    @ApiUnauthorizedResponse({ description: 'Not authorized to verify token.' })
+    @ApiForbiddenResponse({ description: 'Insufficient permissions to verify token.' })
+    @ApiNotFoundResponse({ description: 'Insufficient permissions to verify token.' })
+    @ApiInternalServerErrorResponse({ description: 'Internal server error while verifying a token.' })
+    @Public()
+    public async verifyToken(
```

Comment:
Should this be public?

Reply:
No it shouldnt :D I changed it in the others as well